package.json

```diff
@@ -46,8 +46,13 @@
 		"postcss-loader": "^7.0.1",
 		"resolve-url-loader": "^5.0.0",
 		"sass": "^1.55.0",
-		"sass-lint": "^1.13.1",
 		"sass-loader": "^13.1.0",
+		"stylelint": "^16.3.1",
+		"stylelint-config-recommended": "^14.0.0",
+		"stylelint-config-recommended-scss": "^14.0.0",
+		"stylelint-config-sass-guidelines": "^11.1.0",
+		"stylelint-config-standard": "^36.0.0",
+		"stylelint-scss": "^6.2.1",
```

Comment:
I just merged the eslint PR - but I realised afterwards that all of these should be dependencies in that package, not here. Can you please move these dependencies to the eslint repo so the config and dependencies are in the same place?